graylog2-server/src/test/java/org/graylog2/indexer/retention/strategies/AbstractIndexRetentionStrategyTest.java

```diff
@@ -71,11 +83,21 @@ public void setUp() throws Exception {
         indexMap.put("index5", Collections.emptySet());
         indexMap.put("index6", Collections.emptySet());
 
+        when(indices.indexCreationDate("index1")).thenReturn(Optional.of(NOW.minusDays(1)));
+        when(indices.indexCreationDate("index2")).thenReturn(Optional.of(NOW.minusDays(3)));
+        when(indices.indexCreationDate("index3")).thenReturn(Optional.of(NOW.minusDays(9)));
+        when(indices.indexCreationDate("index4")).thenReturn(Optional.of(NOW.minusDays(10)));
+        when(indices.indexCreationDate("index5")).thenReturn(Optional.of(NOW.minusDays(11)));
+        when(indices.indexCreationDate("index6")).thenReturn(Optional.of(NOW.minusDays(15)));
```

Comment:
Those are sorted in the opposite order in which they would be created in practice.
The oldest index should have the lowest number, not the other way around.

```suggestion
        when(indices.indexCreationDate("index6")).thenReturn(Optional.of(NOW.minusDays(1)));
        when(indices.indexCreationDate("index5")).thenReturn(Optional.of(NOW.minusDays(3)));
        when(indices.indexCreationDate("index4")).thenReturn(Optional.of(NOW.minusDays(9)));
        when(indices.indexCreationDate("index3")).thenReturn(Optional.of(NOW.minusDays(10)));
        when(indices.indexCreationDate("index2")).thenReturn(Optional.of(NOW.minusDays(11)));
        when(indices.indexCreationDate("index1")).thenReturn(Optional.of(NOW.minusDays(15)));
```